backend/init/lang/lang.go

```diff
@@ -127,7 +127,7 @@ func downloadLangFromRemote(fileOp files.FileOp) {
 func downloadGeoFromRemote(fileOp files.FileOp, targetPath string) {
 	_ = os.MkdirAll(path.Dir(targetPath), os.ModePerm)
 	pathItem := fmt.Sprintf("%s/geo/GeoIP.mmdb", global.CONF.System.RepoUrl)
-	if err := fileOp.DownloadFileWithProxy(pathItem, targetPath); err != nil {
+	if err := fileOp.DownloadFile(pathItem, targetPath); err != nil {
 		global.LOG.Errorf("download geo ip failed, err: %v", err)
 		return
 	}
```

Comment:
There are no irregularities or significant performance issues detected in the provided code snippet. The changes to `downloadLangFromRemote` and `downloadGeoFromRemote` functions from using `DownloadFileWithProxy` to simply `DownloadFile` are straightforward and should not introduce new bugs or require further optimizations.

The only change made here is updating the method signature of calling the `fileOp.DownloadFile` function with an additional argument representing the URL instead of a proxy URL. This simplifies the call since we don't need to pass the proxy configuration anymore.